eos/models/standard-model.cc

```diff
@@ -278,16 +278,17 @@ namespace implementation
         for (int i = 0 ; i < 10 ; ++i)
         {
             m_b_MSbar = m_b_msbar(m_b_pole);
-            double next = QCD::m_q_pole(m_b_MSbar, alpha_s(m_b_pole), 5.0);
+            // TODO: adjust alpha_s, and m_b_msbar to 1-loop as well...
```

Comment:
While inconsistent, I would keep ``alpha_s`` and ``m_b_MSbar`` at their current precision and only truncate the loop order in the scheme conversion. This has only a negligible impact on the numerical value.